openapi/aro_hcp/v1alpha1/openapi.json

```diff
@@ -2470,6 +2474,15 @@
           "unknown"
         ]
       },
+      "ClusterImageRegistry": {
+        "description": "ClusterImageRegistry represents the image registry capability for an OpenShift cluster.\nBy default, the image registry is enabled if not explicitly specified.",
```

Comment:
As defined in the corresponding DDR, we need to specify that this is immutable. Same for the state attribute.

The image registry cannot be enabled if it it was disabled during creation. The image registry cannot be disabled if it was enabled during creation. (we do not need to add this paragraph to the doc, just trying to be explicit with explaining the behavior in this PR)